test/neovim/test-runner-ci.sh

```
@@ -0,0 +1,218 @@
+        if test_signal'event then
+            data <= (others => '0');
+        end if;
+    end process;
+end architecture;
+EOF
+}
+
+# Create simple test script that exits cleanly
+create_simple_test() {
+    local test_name="$1"
+    local test_content="$2"
```

Shellcheck (0.10.0)

[warning] test_content appears unused. Verify use (or export if used externally).

(SC2034)